bn_testing/dags.py

```diff
@@ -89,6 +92,7 @@ def _generate_dag(self):
         # Select random edges within the groups
         for group, nodes in self.groups.items():
 
+            # combinations will only give you (A,B) but but not (B,A), is that intended ?
```

Comment:
combinations will only give you (A,B) but but not (B,A), is that intended ? I guess this to avoid cycles by design ?!

Reply:
That's intended. The reason is that the groups are (by design) ordered in a sequential fashion. Will add a comment in the documentation!